_posts/2023-03-22-cockpit-288.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Cockpit 288
+author:
+date: '2023-03-22'
+tags: cockpit, machines, podman
+slug: cockpit-288
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 288, cockpit-machines 286, and cockpit-podman 65:
+
+
+## Metrics: New design with more info in default view
+
+The PCP metrics graphs have been redesigned so as to show by default more data in a single page view. 
+ 
+![screenshot of new design with more info in default view]({{ site.baseurl }}/images/288-metrics-new-design-with-more-info-in-default-view.png)
+![screenshot of new design with more info in default view]({{ site.baseurl }}/images/288-metrics-new-design-with-more-info-in-default-view-2.png)
+
+## API: Drop cockpit.dbus.publish() and .meta()
+
+The `cockpit.js` library dropped these two functions, which were meant to create a D-Bus server object from inside a Cockpit page. No [known Cockpit project](https://cockpit-project.org/applications.html) has ever used this, and we are not aware of a good use case. If you are using this API, please [let us know](https://github.com/cockpit-project/cockpit/issues)!
+
+## Accounts: Show shell and home directory
+
+The account's page now shows user's home directory and default shell.
+
+![223722047-2885b1d2-2bbb-4694-8135-398d51a79387]({{ site.baseurl }}/images/288-accounts-show-shell-and-home-directory.png)
+
+## Services: Pinned units need to be re-done
+
+The data storage for "pinned" systemd units has changed. After upgrading to this Cockpit version, you need to re-pin units.
+
+## Allow specifying UID during account creation
+
+UID should be a integer bigger or same than 1000. There are use cases
```

Comment:
I think explaining the case with non-unique id is an overkill for release notes.